### ISSUE TITLE
Add Dark Mode with a Toggle button

### ISSUE THREAD

**Issue by Nikhil1017** [ZeroOctave/resource-gallery#8]
Add Button with an Icon or Toggle bar in the top right section of navbar nearby to the Contribute to ReGala button.

**Comment by Astrodevil** [comment]
make sure to implement dark theme properly to sync with the whole page UI and colours.

**Comment by Nikhil1017** [comment]
Yes, I wil definitely work on it  and maybe if you want I can show you how it looks by sending it to one of your social media accounts, so we can make changes if needed.

**Comment by Nikhil1017** [comment]
I have send you a request on linkedin

**Comment by Astrodevil** [comment]
You can share screenshots of final changes in PR, we will discuss there if any change will be needed. Continue the work.

**Comment by parasss19** [comment]
### Hi @Astrodevil can I work on this issue as this issue is not completed since last week
###  I already work on this part and contributed successfully.

## Sample Work( Its just a sample and i will make it more better and u can also provide color suggestion)

https://github.com/ZeroOctave/resource-gallery/assets/101205226/46e5abf5-eb21-4637-8ba3-49358071f8d8

**Comment by sagnik-p** [comment]
This issue has been inactive for a long time
I would like to work on this issue
Please assign
